filament/backend/src/vulkan/VulkanBinder.cpp

```diff
@@ -260,8 +260,17 @@ bool VulkanBinder::getOrCreatePipeline(VkPipeline* pipeline) noexcept {
     pipelineCreateInfo.pDepthStencilState = &mPipelineKey.rasterState.depthStencil;
     pipelineCreateInfo.pDynamicState = &dynamicState;
 
+    // Filament assumes consistent blend state across all color attachments.
```

Comment:
oh yeah, interesting one -- I looked into it once, it's actually a GLES limitation. I believe this is lifted in ES3.2